.agents/skills/webawesome/references/components/line-chart.md

```
@@ -0,0 +1,515 @@
+
+To import this component for React 18 or below, use the following code:
+
+```js
+import WaLineChart from '@awesome.me/webawesome/dist/react/line-chart/index.js';
+```
+
+### Slots
+
+| Name | Description |
+| --- | --- |
+| (default) | \`
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)